app/main/templates/review-submission.html

```diff
@@ -32,69 +34,61 @@ <h1 align="center">Review and Submission</h1>
+                    <div class="col-md-5 form-group" ng-class="{ 'has-error' : submissionForm.print.$invalid && submitted}">
+                      <label>Print Name :</label>
+                      <input type="name" placeholder="Print Name" name="print" class="form-control" ng-model="application.request_content.review.printedName" required>
+                      <p ng-show="submissionForm.print.$invalid && submitted" class="help-block">Please print your name</p>
                     </div>
+                <div class="col-md-4 form-group" ng-class="{ 'has-error' : submissionForm.date.$invalid && submitted}">
+                  <label>Date : </label>
+                  <p class="input-group">
+                  <input id="date" type="text" class="form-control" uib-datepicker-popup="{{format}}" ng-model="application.request_content.review.signatureDate"
+                      is-open="popup1.opened" name="date" datepicker-options="dateOptions" ng-required="true" close-text="Close"
+                      alt-input-formats="altInputFormats" placeholder="Date" required>
+                    <span class="input-group-btn">
+                      <button type="button" class="btn btn-default" ng-click="open1()">
+                        <i class="glyphicon glyphicon-calendar"></i>
+                      </button>
+                    </span>
+                    <p ng-show="submissionForm.date.$invalid && submitted" class="help-block">Please select a date</p>
+                  </p>
                 </div>
-            </div>
-            <br>
-            <div class="col-md-8">
-                <label>Signature</label>
-                <signaturepad/>
-            </div>
-
-            <br>
-
-        </div>
+          </div>
+        </form>
+     </div>
+    <div class="row">
+       <div class="col-md-10">
+            <signaturepad signature="application.request_content.review.signature" />
+      </div>
     </div>
+  </div>
+
+  <div class="form-nav">
+    <button ui-sref="home" class="button inverse w-button" id="save">Save for Later</button>
+    <button  class="button w-button" ng-click="submit(submissionForm.$valid)">Submit</button>
+  </div>
+</div>
 </div>
-<!-- Pager -->
-<div class="row">
-    <div class="pull-right">
-        <nav>
-            <ul class="pager">
-                <li class="previous"><a href="#"><span class="glyphicon glyphicon-menu-left"
-                                                       aria-hidden="true"></span> Previous</a></li>
-                <li class="next"><a href="#">Next <span class="glyphicon glyphicon-menu-right"
-                                                        aria-hidden="true"></span></a></li>
-            </ul>
-        </nav>
-    </div>
-</div> <!-- /.row -->
 
+</div>
 
-<hr>
-<footer class="margin-tb-3">
-    <div class="row">
-        <div class="col-lg-12">
-            <p>Copyright &copy; AZ Hackathon 2016</p>
+    <script type="text/ng-template" id="myModalContent.html">
```

Comment:
Rename this from "myModalContent" to something more descriptive; this will need to be updated in a few other places too